python/its-quadkeys/its_quadkeys/quadkeys.py

```diff
@@ -383,11 +390,13 @@ def optimise(self):
                     continue
 
                 # Are this QuadKey and the following three making a super
-                # QuadKey? I.e. do we have root0, root1, root2, and root3?
+                # QuadKey? I.e. do we have 'root0', 'root1', 'root2', and
+                # 'root3', with an non-empty 'root' (issue github#130)
```

Comment:
> issue GitHub#130

to activate the link on my side

Reply:
I think `#130` should also be enough. I'll try to push with that.

Reply:
I tested both `#130` and `GitHub#130`, and neither created a link when browsing the source code, so I left it as just `#130` in the code, as it is explicitly stated that it is an issue, and it is implicitly an issue on GitHub.